ros/angel_utils/scripts/bag_extractor.py

```diff
@@ -82,14 +118,20 @@ def __init__(self):
         if self.extract_head_pose_data:
             self.msg_type_to_handler_map[HeadsetPoseData] = self.handle_head_pose_msg
         if self.extract_hand_pose_data:
-            self.msg_type_to_handler_map[HandJointPosesUpdate] = self.handle_hand_pose_msg
+            self.msg_type_to_handler_map[
+                HandJointPosesUpdate
+            ] = self.handle_hand_pose_msg
         if self.extract_spatial_map_data:
             self.msg_type_to_handler_map[SpatialMesh] = self.handle_spatial_mesh_msg
+        if self.extract_annotation_event_data:
+            self.msg_type_to_handler_map[
+                AnnotationEvent
+            ] = self.handle_annotation_event_msg
 
         # Top level data folder
         self.num_total_msgs = 0
         self.data_folder = self.bag_path + "_extracted/"
-        if not(os.path.exists(self.data_folder)):
+        if not (os.path.exists(self.data_folder)):
```

Comment:
Given the reformatting, the parentheses around the os function are extraneous. Same for similar instances below.
```suggestion
        if not os.path.exists(self.data_folder):
```